### PR TITLE
Final Project

### PR DESCRIPTION
**Pull request by BScottHenderson**
Not really working but I just ran out of time.  Submitting anyway on the theory that something is better than nothing.

Great class - I learned a lot.

### CODE REVIEW

**Comment by jamdon** [comment]
You still have time to work on this if you need. The due date is the 18th. I can look at your code and provide feedback if you want.

File: public/js/views/AllRatingsView.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* global TFT */
+
+"use strict";
+
+var Backbone = require("backbone");
+var RatingItemView = require("./RatingItemView");
+
+module.exports = Backbone.View.extend({
+	className: "allRatingsList",
+	template: TFT.ratings,
```
Comment (jamdon) [inline public/js/views/AllRatingsView.js:10]: your template file is allRatings, not ratings, and this is causing an error that breaks your application as soon as you navigate to the ratings in your app

File: public/js/views/NachosByRatingView.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	_children: [],
+	initialize: function (options) {
+		if (options) {
+			this.rating = options.rating;
+		}
+	},
+	render: function () {
+		var data = { location: this.location };
+
+		this.$el.append(this.template(data));
```
Comment (jamdon) [inline public/js/views/NachosByRatingView.js:21]: This template wants the rating, not the location data.

**Comment by jamdon** [comment]
I really only found two issues in your code and I've left inline comments about them. I can grade it as-is if you want, or you can correct them first if you want. Let me know.

**Comment by BScottHenderson** [comment]
Thanks very much for taking a look. I would fix the issues if I could but I
am out of town through the weekend without my laptop. I really got a lot
out of the class in spite of my less than stellar performance on the final
project.

TL;DR Please grade as is.

**Comment by jamdon** [comment]
👍
